[2024-09-27T15:51:50.850+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-27T15:51:50.864+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_costos_ventas.load_data manual__2024-09-27T15:51:44.442856+00:00 [queued]>
[2024-09-27T15:51:50.871+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_costos_ventas.load_data manual__2024-09-27T15:51:44.442856+00:00 [queued]>
[2024-09-27T15:51:50.871+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-27T15:51:50.881+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_data> on 2024-09-27 15:51:44.442856+00:00
[2024-09-27T15:51:50.889+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=83) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-27T15:51:50.890+0000] {standard_task_runner.py:72} INFO - Started process 85 to run task
[2024-09-27T15:51:50.890+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_costos_ventas', 'load_data', 'manual__2024-09-27T15:51:44.442856+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/RPA_dag.py', '--cfg-path', '/tmp/tmpnh8harur']
[2024-09-27T15:51:50.891+0000] {standard_task_runner.py:105} INFO - Job 52: Subtask load_data
[2024-09-27T15:51:50.927+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_costos_ventas.load_data manual__2024-09-27T15:51:44.442856+00:00 [running]> on host 32d53c0d21c3
[2024-09-27T15:51:50.993+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_costos_ventas' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-27T15:51:44.442856+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-27T15:51:44.442856+00:00'
[2024-09-27T15:51:50.995+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-27T15:51:50.995+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-27T15:51:50.996+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2024-09-27 15:51:50.864918+00:00
[2024-09-27T15:51:50.996+0000] {logging_mixin.py:190} INFO - Dag name:etl_costos_ventas and current dag run status:running
[2024-09-27T15:51:50.996+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-27T15:51:51.050+0000] {logging_mixin.py:190} INFO - Archivo Excel corregido generado con Ã©xito.
[2024-09-27T15:51:51.051+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-27T15:51:51.062+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-27T15:51:51.062+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_costos_ventas, task_id=load_data, run_id=manual__2024-09-27T15:51:44.442856+00:00, execution_date=20240927T155144, start_date=20240927T155150, end_date=20240927T155151
[2024-09-27T15:51:51.072+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-27T15:51:51.073+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-27T15:51:51.073+0000] {logging_mixin.py:190} INFO - Dag name:etl_costos_ventas queued_at:2024-09-27 15:51:44.453966+00:00
[2024-09-27T15:51:51.074+0000] {logging_mixin.py:190} INFO - Task hostname:32d53c0d21c3 operator:PythonOperator
[2024-09-27T15:51:51.104+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-27T15:51:51.118+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-27T15:51:51.120+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
